release.sh

```diff
@@ -5,22 +5,18 @@ set -euf -o pipefail
 read -p 'Have you read DEVELOPMENT.md? [y/N] ' -n 1 -r
 echo
 case "$REPLY" in
-    Y|y) ;;
```

Comment:
I don't know why this is still getting formatted on file save when I've disabled format-on-save for all languages but go 😢

Reply:
🤷‍♂️, honestly 😄